docs/capabilities/read_only_tenant_mode.md

```diff
@@ -0,0 +1,56 @@
+# Read Only Tenant Mode
+
+There are two distinct functionalities for "read-only" access in Dashboards. One of them is associated with roles and one is associated with tenants. Regarding the first one, the Dashboards Security plugin contains a feature of hiding all plugin navigation links except Dashboards and Visualizations when the logged-in user has a certain role. The second one is limiting Dashboards access rights via assigning a specific role to a tenant (therefore, making a tenant read-only). Due to past issues and the deprecation of the first functionality, using read-only tenants is now the recommended way to limit users' access to Dashboards.
```

Comment:
This paragraph is a little hard to understand, for example whats the difference between a role and a tenant? This might be due me not knowing enough about the security plugin and how it works, but i assume other developers reading this also might be. Here are some suggestios for making this easier to understand:
1. Clarify the distinction between `role` and `tenant`
2. Provide a simple example for each